### DISCUSSION TITLE
Support for a few more non-bird sounds

### DISCUSSION THREAD

**Discussion by forthrin** [birdnet-team/BirdNET-Analyzer#355] [Q&A]
Noticed some non-bird sounds are identified, like "dog", "siren", etc. 

Assumed because they randomly inhabit bird recordings, and may prove interesting or function as markers.

Is it possible to add support for a few other background noises?

Nature:
- Cat (meowing)
- Rain
- Thunder

Human:
- Music (just "any" music, not going into specifics)
- Applause
- Laughter

**Comment by kahst** [comment]
We will review the non-bird classes in future versions, and will likely merge some and create some new ones. Rain and thunder are currently combined as "Environmental". However, this class is treated as "0" during training and will likely never trigger a detection. The challenge (as always) is training data. We used AudioSet before, but will likely switch to more reliable sources in the future. You'll still be able to add more classes through retraining if there is a need.

**Comment by forthrin** [comment]
Aren't Rain and Applause sort of "white noise" that can be recognized without advanced training?

Meowing and Thunder is also quite generic (quite identical from case to case).

Laughter is admittedly a bit more complex and varying.

Music even more so, but someone must surely have made a "This segment is music" boolean by this point.

What's the current ETA on new features relating to this?
